tests/unit/test_session.py

```diff
@@ -3555,7 +3563,10 @@ def test_endpoint_from_production_variants_with_sagemaker_config_injection_parti
     sagemaker_session.sagemaker_config = SAGEMAKER_CONFIG_ENDPOINT_CONFIG
 
     sagemaker_session.sagemaker_client.describe_endpoint = Mock(
-        return_value={"EndpointStatus": "InService"}
+        return_value={
+            "EndpointStatus": "InService",
+            "EndpointArn": "arn:aws:sagemaker:us-west-2:123456789012:endpoint/test",
```

Comment:
Can endpoint arn be empty?

Basically is this change backward compatible?

Reply:
Yes, endpoint arn will always be set in the response of describe endpoint api call.

Reply:
I think we need you need to add : feat, and/or breaking to the overall commit since the response contract will change 

https://github.com/aws/sagemaker-python-sdk/blob/d5dfe2f2085e7a7e2653167394abee8405da1d4f/CONTRIBUTING.md?plain=1#L131-L133